system/events/mumble.py

```diff
@@ -396,6 +396,62 @@ class UserRecordingToggle(UserStateToggleEvent):
     pass
 
 
+class UserStats(MumbleEvent):
+    """
+    User stats updated
+
+    user: User whose stats have been updated
+    """
+
+    def __init__(self, caller, user):
+        """
+        Initialise the event object.
+        """
+
+        self.user = user
+
+        super(UserStats, self).__init__(caller)
+
+
+class UserRegisteredEvent(MumbleEvent):
```

Comment:
Ah, yeah. I had `UserRegistrationEvent` before, but changed since it didn't match the sub-classes. Either way, nothing should be using the base class anyway.